bool electronic_relay
bool motor_relay
bool cable_relay
---
bool success